Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -221,9 +221,21 @@ protected virtual void Dispose(bool disposing)
         /// </summary>
         private void InitializeForFrameworkEventSource()
         {
-#if !NET40
-            FrameworkHttpProcessing frameworkHttpProcessing = new FrameworkHttpProcessing(this.telemetryConfiguration, DependencyTableStore.Instance.WebRequestCacheHolder, this.SetComponentCorrelationHttpHeaders, this.ExcludeComponentCorrelationHttpHeadersOnDomains, this.EffectiveProfileQueryEndpoint);
-            this.httpDesktopDiagnosticSourceListener = new HttpDesktopDiagnosticSourceListener(frameworkHttpProcessing);
+#if NET45
+            DesktopDiagnosticSourceHttpProcessing desktopHttpProcessing = new DesktopDiagnosticSourceHttpProcessing(
```

Comment:
it's [here](https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/4f7d1518545b8a6e88d7455b49fb0bc0887e0e8d/Src/DependencyCollector/Shared/Implementation/FrameworkHttpProcessing.cs#L44)

In runtime, we do not know what happens first: initialization of Http Desktop DiagnosticSource or initialization of the DependencyTrackingTelemetryModule, so we keep listening to both, but ignoring the EventSource if DiagnosticSource is active.